src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -255,6 +255,21 @@ <h3 id="history">
 
     {% if is_admin %}
     <ul class="admin_only">
+      {% if not version.needs_human_review %}
```

Comment:
Mmm, yes, I did wonder if that was needed. But I thought the normal way to mark it as not needing human review any longer was to just review it... otherwise there would be, in effect, two ways to change the version to a state that didn't need review.

Thoughts @wagnerand ?